api/src/main/java/org/eclipse/microprofile/faulttolerance/Fallback.java

```diff
@@ -36,6 +36,17 @@
  * <li>If fallbackMethod is specified, invoke the method specified by the fallbackMethod on the same class.</li>
  * <li>If both are specified, the {@link org.eclipse.microprofile.faulttolerance.exceptions.FaultToleranceDefinitionException} must be thrown.</li>
  * </ol>
+ * <p>
+ * When a method returns and the Fallback policy is present, the following rules are applied:
+ * <ol>
+ * <li>If the method returns normally (doesn't throw), the result is simply returned.
+ * <li>Otherwise, if the thrown object is assignable to any value in the {@link #skipOn()} parameter, the thrown object will be rethrown.
+ * <li>Otherwise, if the thrown object is assignable to any value in the {@link #applyOn()} parameter, 
```

Comment:
maybe we can rewrite this to "Otherwise if the thrown exception is part of the parameter list {@link #applyOn()} parameter, the Fallback policy, detailed above, will be applied."

Reply:
dtto

Reply:
See the above comments